### PR TITLE
#185923789 : Using ILogger for logging

### PR DESCRIPTION
**Pull request by qmjava**
Pull Request review Checklist:
 - [x] `Follows` guidelines for `public` repo ?
 - [x] `README` updated, if required ?
 - [x] `Release Version` updated ?
 - [x] `User Agent` Version updated ?
 - [x] `Includes` tests conducted and `test result` in the ticket and/or QA doc ?
 - [x] `Does NOT` include code from others?  (Unintended consequence of a bad merge) ?

### CODE REVIEW

File: Moesif.Middleware/NetCore/Helpers/LoggerHelper.cs [hunk 1]
```diff
@@ -21,36 +21,38 @@ namespace Moesif.Middleware.NetCore.Helpers
...
-            }
+            _logger = logger;
+        }
+        public void LogDebugMessage(bool debug, String msg)
+        {
+            _logger.LogDebug(msg);
         }
 
-        public static string GetConfigStringValues(Dictionary<string, object> moesifOptions, String configName, string defaultValue)
+        public string GetConfigStringValues(Dictionary<string, object> moesifOptions, String configName, string defaultValue)
```
Comment (CodeYouMust) [inline Moesif.Middleware/NetCore/Helpers/LoggerHelper.cs:34]: is the removal of `static` going to cause any backwards compatibility issue.. meaning dont think this should be used directly outside of this library?

Comment (qmjava) [reply Moesif.Middleware/NetCore/Helpers/LoggerHelper.cs:34]: Nope.

File: Moesif.Middleware/Moesif.Middleware.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```
Comment (CodeYouMust) [inline Moesif.Middleware/Moesif.Middleware.csproj:4]: care to share background on what necessary to change this? the `README.md` still specifies its supported on `4.5x` - do we no longer support.. should then update readme too.

Comment (qmjava) [reply Moesif.Middleware/Moesif.Middleware.csproj:4]: yes.  It is due to this ticket.
https://github.com/microsoft/ApplicationInsights-dotnet/issues/1312

Comment (CodeYouMust) [reply Moesif.Middleware/Moesif.Middleware.csproj:4]: if minimum support is changing.. please update `README.md` in this repo too to remove `4.5`.

**Review by praves77 — APPROVED** [review]

**Pull request merged**